src/03-filter.js

```diff
@@ -13,21 +13,27 @@ const exampleSongData = require("../data/songs");
  * @param {Object[]} songs - An array of songs. See the song data for more.
```

Comment:
If you're returning the element without any changes from the callback you pass to map, than you don't need to use map at all